cmd/acra-server/common/listener.go

```diff
@@ -38,15 +39,16 @@ type SServer struct {
 	config                *Config
 	listenerACRA          net.Listener
 	listenerAPI           net.Listener
-	fddACRA               uintptr
```

Comment:
why remove it? it used for sighup handler and fork

Reply:
> why remove it? it used for sighup handler and fork

This ones are unused. See [this commit](https://github.com/cossacklabs/acra/pull/415/commits/ef26077b7ce868a541e16fa4d347638ad97ab48a) which removes all places where `fddACRA` and `fdAPI` are used: they are set to constant values of fds and never used again.

Actual handling of these fds [happens here](https://github.com/cossacklabs/acra/blob/b2cb4f34aa18edb0eb95d18aa641bf7daedbd546/cmd/acra-server/acra-server.go#L374-L396): the fd values are extracted from `net.Listener` objects and passed to the new process after standard fds, as fds 3 and 4, [aka `descriptorAcra` and `descriptorAPI`](https://github.com/cossacklabs/acra/blob/b2cb4f34aa18edb0eb95d18aa641bf7daedbd546/cmd/acra-server/acra-server.go#L68-L69).